stonesoup/models/measurement/nonlinear.py

```diff
@@ -90,9 +90,7 @@ def rvs(self, num_samples=1, **kwargs):
             return rvs_vectors.view(Matrix)
 
 
-class NonLinearGaussianMeasurement(MeasurementModel,
-                                   NonLinearModel,
-                                   GaussianModel):
+class NonLinearGaussianMeasurement(MeasurementModel, NonLinearModel, GaussianModel, ABC):
```

Comment:
These are already abstract, so no need to inherit from `ABC` again.

Reply:
Do you say "they are already abstract" because they have `@abstractmethod`s?

If so then I agree, but thought this was clearer to see if it was in the class definition as well. Happy to revert though.

Reply:
The `BaseMeta` class inherits from `ABCMeta`, which `Base` and then all components are based from.

Reply:
Yes, only reason is for readability then. If not useful (which it probably isn't) then I'll take it out again.

OTOH: to quote SO :)
> and also readability. Inheriting from ABC makes it clear to your readers what you're up to.

Reply:
I'm easy on this one.

Reply:
I think we have decided to leave this as is.